Source/Spray/SprayParticles.H

```diff
@@ -242,6 +245,29 @@ public:
       pele::physics::TransportType> const* ltransparm,
     const amrex::Real spray_cfl_lev);
 
+  /// \brief Update particle data and solve gas-phase source terms (to be used with manifold model)
```

Comment:
I don't think this version gets defined or called anywhere - can you delete the declaration from the header?